src/Microsoft.AspNet.Razor.Runtime/TagHelpers/ITagHelper.cs

```diff
@@ -17,6 +17,16 @@ public interface ITagHelper
         int Order { get; }
```

Comment:
Unfortunately `<remarks>` don't show up in IntelliSense and @Eilon loves IntelliSense. Suggest using this instead of the vaguer "execution order" in the `<summary>`.